src/junglescout/session.py

```diff
@@ -1,15 +1,15 @@
 import urllib.parse
 from typing import Dict, Optional
 
-import requests
+import httpx
 
 from junglescout.models.parameters import ApiType
 
 
-class Session(requests.Session):
+class BaseSession():
```

Comment:
Consider making `BaseSession` an abstract base class, since it is now just responsible for defining the `Session` interface.